cogs/base_cog.py

```diff
@@ -5,17 +5,45 @@
 from custom_errors  import InvalidTypeError
 
 class BaseCog(commands.Cog):
-    def __init__(self, bot, category):
+    def __init__(self, bot):
         self.bot = bot
-        self.category = category
-
         self.data_path = get_json_path("data")
 
-    # Get data from JSON file
     async def get_data(self):
+        """
+            Return the data stored inside the ``data.json`` file.
+
+            Returns
+            -------
+            json_data: :class:`dict`
+                the JSON object.
+        """
+
         return await get_json_data(self.data_path)
 
     async def get_guild_data(self, guild, data=None):
+        """
+            Return the guild data stored inside ``data``. If ``data`` is not given,
+            retrieve the data from the ``data.json`` file.
+
+            Parameters
+            ----------
+            guild: :class:`discord.Guild`
+                a discord guild.
+            data: :class:`dict, optional`
+                the JSON object.
```

Comment:
Multiline code suggestion -- same as the normal code suggestion, just select multiple lines to comment

```suggestion
    # hiiii
```